kubernetes/cmsweb/scripts/deploy-srv-wmcore.yaml

```diff
@@ -0,0 +1,190 @@
+echo ${srv[*]}
+
+
+cmsweb_image_tag=:$1
+
+if [ $# == 2 ]; then
+	env=$2
+fi
+
+cmsweb_env=k8s-$env
+cmsweb_log=logs-cephfs-claim-$env
+
+if [[ "$cluster_name" == *"testbed"* ]] ; then
+        if [[ "$env" != "preprod" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name" == *"prod"* ]] ; then
+        if [[ "$env" != "prod" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name"  == *"cmsweb-auth"* ]] ; then
+        if [[ "$env" != "auth" ]] ; then
+        check=false
+        fi
+fi
+
+if [[ "$cluster_name"  == *"cmsweb-test1" ]] ; then
+        if [[ "$env" != "test1" ]] ; then
+        check=false
+        fi
+fi
+
+if [[ "$cluster_name"  == *"cmsweb-test2" ]] ; then
+        if [[ "$env" != "test2" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name"  == *"cmsweb-test3" ]] ; then
+        if [[ "$env" != "test3" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name"  == *"cmsweb-test4" ]] ; then
+        if [[ "$env" != "test4" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name"  == *"cmsweb-test5" ]] ; then
+        if [[ "$env" != "test5" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name"  == *"cmsweb-test6" ]] ; then
+        if [[ "$env" != "test6" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name"  == *"cmsweb-test7" ]] ; then
+        if [[ "$env" != "test7" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name"  == *"cmsweb-test8" ]] ; then
+        if [[ "$env" != "test8" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name"  == *"cmsweb-test9" ]] ; then
+        if [[ "$env" != "test9" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name"  == *"cmsweb-test10" ]] ; then
+        if [[ "$env" != "test10" ]] ; then
+        check=false
+        fi
+fi
+if [[ "$cluster_name"  == *"cmsweb-test11" ]] ; then
+        if [[ "$env" != "test11" ]] ; then
+        check=false
+        fi
+fi
+
+if [[ $check == false ]] ; then
+
+        echo "The environment and config did not match. Please check."
+        exit 1;
+fi
+
+
+for i in "${srv[@]}"
+do
+
+        tmpDir=/tmp/$USER/k8s/srv
+
+        # use tmp area to store service file
+        if [ -d $tmpDir ]; then
+        rm -rf $tmpDir
+        fi
+        mkdir -p $tmpDir
+        cd $tmpDir
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/kubernetes/cmsweb/services/$i.yaml
+
+        # check that service file has imagetag
+        if [ -z "`grep imagetag $i.yaml`" ]; then
+         echo $i
+         echo "unable to locate imagetag in $i.yaml"
+        exit 1
+        fi
+        echo "The downloaded and newly generated service manifest files are available at $tmpDir"
+        echo "Using Environment: $cmsweb_env"
+
+        # replace imagetag with real value and deploy new service
+        if [ "$cmsweb_env" == "k8s-prod" ] ; then
```

Comment:
@arooshap 

Please add OR condition of `k8s-preprod` as well as #PROD# prefix is replaced in production and testbed clusters. You also need to remove the check from below regarding `k8s-preprod`